mlflow/tracking/default_experiment/databricks_notebook_experiment_provider.py

```diff
@@ -1,5 +1,14 @@
+from mlflow.exceptions import MlflowException
+from mlflow.protos import databricks_pb2
+from mlflow.tracking import MlflowClient
 from mlflow.tracking.default_experiment.abstract_context import DefaultExperimentProvider
 from mlflow.utils import databricks_utils
+from mlflow.utils.mlflow_tags import (
+    MLFLOW_EXPERIMENT_SOURCE_TYPE,
+    MLFLOW_EXPERIMENT_SOURCE_ID,
+)
+
+_resolved_repo_notebook_experiment_id = None
```

Comment:
Can `_resolved_repo_notebook_experiment_id` be a static class variable instead of at module level? It has nothing to do with `DatabricksNotebookExperimentProvider`.